src/App.tsx

```diff
@@ -1,60 +1,107 @@
-import React from 'react';
+import React, { useContext, useMemo, useState } from 'react';
 import 'bulma/bulma.sass';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-import classNames from 'classnames';
+import cn from 'classnames';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { AppContext } from './AppContext';
+import { User } from './types/User';
+import * as service from './api/api';
 
 export const App: React.FC = () => {
+  const {
+    isUserError,
+    setIsUserError,
+    selectedUser,
+    posts,
+    setPosts,
+    selectedPost,
+    setSelectedPost,
+  } = useContext(AppContext);
+
+  const [isLoading, setIsLoading] = useState(false);
+  // const [isError, setIsError] = useState(false);
```

Comment:
```suggestion
  // const [isError, setIsError] = useState(false);
```
Remove commented code